src/pocket/PocketApiGatewayLambdaIntegration.ts

```diff
@@ -69,7 +78,10 @@ export class PocketApiGateway extends Resource {
           redeployment: Fn.sha1(
             Fn.jsonencode({
               resources: routeDependencies.map((d) => d.id),
-              config: JSON.stringify(config),
+              config: JSON.stringify({
```

Comment:
I hate how tricky this is in general.

Reply:
Not your code, AWS API Gateway behavior

Reply:
ya we have to recursively make sure there are no list in the `config` object.